go.mod

```diff
@@ -6,11 +6,10 @@ require (
 	cloud.google.com/go/pubsub v1.0.1 // indirect
 	github.com/BurntSushi/toml v0.3.1
 	github.com/aws/aws-sdk-go v1.23.18
-	github.com/gordonklaus/ineffassign v0.0.0-20200809085317-e36bfde3bb78 // indirect
```

Comment:
I don't think we want these changes to go.mod and go.sum since you didn't change any dependencies

Reply:
I reverted them.